app/Models/CombinedFragment.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class CombinedFragment extends Model {
+
+	protected $table = 'combined_fragments';
+	public $timestamps = true;
+
+	public function lifelineCombinedFragment()
+	{
+		return $this->hasMany('LifelineCombinedFragments');
```

Comment:
toto skončí s ModelNotFoundException lebo model \LifelineCombinedFragments neexistuje len \App\Models\LifelineCombinedFragments takže zmeniť všade s použitím class konštanty teda LifelineCombinedFragments::class